util.c

```diff
@@ -2303,21 +2306,27 @@ void manage_fork_fds(int close_all)
 {
 	DIR *dir;
 	struct dirent *dirent;
+	int fd;
 
 	close(0);
-	open("/dev/null", O_RDWR);
+	fd = open("/dev/null", O_RDWR);
+	if (fd < 0)
```

Comment:
This is a way to redirect streams to `/dev/null`, if `/dev/null` is not achievable, we have to close them (I mean 0, 1, 2).
If should not be a real case but we need to handle it.